tests/std/test.lst

```diff
@@ -157,6 +157,7 @@ tests\Dev11_1158803_regex_thread_safety
 tests\Dev11_1180290_filesystem_error_code
 tests\GH_000457_system_error_message
 tests\GH_000545_include_compare
+tests\GH_000685_condition_variable_any

```

Comment:
I just realized that we're accumulating inconsistency here. All of our Dev11, VSO, etc. tests refer to bug numbers (in internal databases). I thought that was going to be our policy for GH tests, so my GH-545 test refers to an issue number. However, the GH-457 test refers to a PR number, as does this GH-685 test.

Since PRs and issues are cross-referenced, I suppose it doesn't matter, and our policy going forward can be "either PR or issue numbers are acceptable". Still, I wanted to mention this as an explicit decision.

Reply:
I think if we're willing to move to accepting PR numbers we should be more explicit about always linking an issue to the PR.

Reply:
Good idea, I've updated https://github.com/microsoft/STL/wiki/Checklist-For-Merging-A-Pull-Request with "Verify that the PR is linked to the issues that it resolves."